projects/bootloader/src/stm32f0xx/jump_to_application.c

```diff
@@ -19,8 +22,17 @@ static noreturn __attribute__((naked)) void prv_perform_jump(uint32_t sp, uint32
 }
 
 void jump_to_application(void) {
-  // TODO(SOFT-413): check that this is safe with a crc of the application code
-  // TODO(SOFT-413): deinitialize any libraries the bootloader uses before doing this
+  BootloaderConfig config = { 0 };
+  config_get(&config);
+
+  // get the computed crc32 code
+  uint32_t computed_crc32 = calculated_application_crc32((uintptr_t)BOOTLOADER_APPLICATION_START,
+                                                         BOOTLOADER_APPLICATION_SIZE);
+
+  if (config.application_crc32 != computed_crc32) {
+    LOG_DEBUG("CRC32 codes do not match returning");
```

Comment:
There needs to be a `\n` at the end of log messages.

Reply:
done